lib/chef/audit/runner.rb

```diff
@@ -45,6 +45,10 @@ def num_total
         RSpec.world.reporter.examples.size
       end
 
+      def exclusion_pattern exclusion_path
+          Regexp.new(exclusion_path.split(/[\\\/]/)[0..2].join("[\\\/]"))
```

Comment:
Was trying to do / or \ or any combination of the two. I didn't think I could use File::SEPERATOR in a regex. Not sure about an escaped space. I changed it now to just look for /lib/chef in the path because we thought this way was too inclusive. So now we don't have to worry about the space.